internal/kube/site/site.go

```diff
@@ -181,9 +184,22 @@ func (s *Site) reconcile(siteDef *skupperv2alpha1.Site, inRecovery bool) error {
 	}
 
 	// 3. deployment
+	size, err := s.sizes.GetSizing(s.site)
+	if err != nil {
+		s.logger.Info("Did not retrieve size for site",
+			slog.String("namespace", s.site.Namespace),
+			slog.String("name", s.site.Name),
+			slog.String("reason", err.Error()),
+		)
+	} else {
+		s.logger.Info("Sizing for site",
```

Comment:
Minor recommendation to log it with Debug as it keeps showing up in the logs every 30 seconds.